### DISCUSSION TITLE
Consulta TPA 2da entrega

### DISCUSSION THREAD

**Discussion by rubinhoemanuel** [dds-jv/foro#426] [TPA - 2024] [answered]
Buenas tardes,

Tengo una consulta respecto al sensor de temperatura, en donde, en la interfaz Action no esta especificado que tipo de dato devuelve el metodo:

executeForTemperature(double temperature)

Desde ya muchas gracias,
saludos.

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola! `executeForTemperature` es un _callback_ que no devuelve nada al ser ejecutado. Su tipo de retorno es `void`. 

¡Saludos!
